elf_binary.cc

```diff
@@ -133,7 +132,18 @@ void ELFBinary::ReadDynSymtab(const std::map<std::string, std::string>& filename
 
         // Get version information coresspoinds to idx
         auto p = GetVersion(idx, filename_to_soname);
-        Elf_Versym v = (versym_) ? versym_[idx] : -1;
+        Elf_Versym v;
+        if (versym_) {
```

Comment:
I want ELFBinary to be just a parser of ELF module and do not want it to interpret the information. I think this analysis should be done in sold.cc or its helper classes.

Reply:
I understood. I moved this procedure to sold.cc.